.drone.yml

```diff
@@ -143,6 +143,13 @@ steps:
   depends_on:
   - package
 
+- name: build-frontend-docs
+  image: grafana/build-container:1.2.27
+  commands:
+  - ./scripts/ci-reference-docs-lint.sh ci
+  depends_on:
+  - build-frontend
```

Comment:
Does it have to run _after_ build-frontend? Can it execute in parallel instead, for speed?

Reply:
yes, it is depending on the packages have been built. So if you remember we added that to the `build-frontend` build step.